packages/azure-http-specs/specs/service/multi-service/main.tsp

```diff
@@ -0,0 +1,59 @@
+import "@typespec/versioning";
+import "@typespec/http";
+import "@typespec/spector";
+
+using Versioning;
+using Http;
+using Spector;
+
+namespace Service.MultiService;
+
+/*
+ * First service definition in a multi-service package with versioning
+ */
+@scenarioService("/service/multi-service/service-a")
+@versioned(VersionsA)
+namespace ServiceA {
+  enum VersionsA {
+    av1,
+    av2,
+  }
+
+  interface AI {
```

Comment:
nit, not sure what `AI` and `BI` means. but this is just a test so OK for me.

Reply:
A interface. Let me refine the naming.